affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/plugin/AffectedModuleTaskType.kt

```diff
@@ -0,0 +1,30 @@
+package com.dropbox.affectedmoduledetector.plugin
+
+/**
+ * For creating a custom task which will be run only if module was affected
+ * just create enum which implements this interface.
+ *
+ * Your enum must override all this variable
```

Comment:
I don't think you need to specify this, you always have to implement all variables in an interface

Reply:
changed - "enum" to "data structure"